digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/oracle/OracleLinearAssetDao.scala

```diff
@@ -161,7 +161,8 @@ class OracleLinearAssetDao(val vvhClient: VVHClient) {
         where a.asset_type_id = $assetTypeId and a.id = $id
         """.as[(Long, Double, Double)].list
 
-    val roadLinksByLinkId = vvhClient.fetchByLinkIds(links.map(_._1).toSet)
+    //TODO This sould be done in DAO object

```

Comment:
This is something we have found when we were developing but didn't want to touch it now. In future developments if we have to do something here we can move it away from the DAO.